resolvers.ts

```diff
@@ -193,27 +194,143 @@ export const resolvers: Resolvers = {
+            time: input?.todoRemove?.time,
+            tissue: input?.where?.sampleType?._in,
+            visibility: input?.todoRemove?.visibility,
+            workflow: input?.todoRemove?.workflow,
+            //  - DiscoveryDataLayer.ts
+            //    await this._collection.fetchDataCallback({
+            limit: input?.todoRemove?.limit,
+            offset: input?.todoRemove?.offset,
+            listAllIds: input?.todoRemove?.listAllIds,
+          }),
+        args,
+        context
+      );
+      if (!workflow_runs?.length) {
+        return [];
       }
+
+      return workflow_runs.map((run): query_samples_items => {
+        const inputs = run.inputs;
+        const qualityMetrics = run.cached_results?.quality_metrics;
+        const sample = run.sample;
+        const sampleInfo = sample?.info;
+        const sampleMetadata = sample?.metadata;
+        return {
+          todoRemove: {
+            status: run.status,
+            startedAt: run.created_at,
+            creationSource: inputs?.creation_source,
+            workflowVersion: {
+              version: run.wdl_version,
+            },
+          },
+          id: sample?.id,
+          name: sampleInfo?.name,
+          notes: sampleInfo?.sample_notes,
+          collectionLocation: sampleMetadata?.collection_location_v2,
+          sampleType: sampleMetadata?.sample_type,
+          waterControl: sampleMetadata?.water_control,
+          hostTaxon: {
+            name: sampleInfo?.host_genome_name,
+          },
+          collection: {
+            name: sample?.project_name,
+            public: Boolean(sampleInfo?.public),
+          },
+          ownerUser: {
+            name: sample?.uploader?.name,
+          },
+          metadatas: {
+            edges:
+              sampleMetadata != null
+                ? Object.entries(sampleMetadata)
+                    .filter(
+                      ([fieldName]) =>
+                        fieldName !== "nucleotide_type" &&
+                        fieldName !== "collection_location_v2" &&
+                        fieldName !== "sample_type" &&
+                        fieldName !== "water_control"
+                    )
+                    .map(([fieldName, value]) => ({
+                      node: {
+                        fieldName,
+                        value: String(value),
+                      },
+                    }))
+                : [],
+          },
+          sequencingReads: {
+            edges: [
+              {
+                node: {
+                  nucleicAcid: sampleMetadata?.nucleotide_type,
+                  protocol: inputs?.wetlab_protocol,
+                  medakaModel: inputs?.medaka_model,
+                  technology: inputs?.technology,
+                  consensusGenomes: {
+                    edges: [
+                      {
+                        node: {
+                          taxon: {
+                            name: inputs?.taxon_name,
```

Comment:
This doesn't exactly match the processing that happens in `processConsensusGenomeWorkflowRun` and `processRawWorkflowRun` -- are there fields that are unused in those old processing functions?

Reply:
There are a few, like from a quick look `userIdWhoInitiatedWorkflowRun` is one. Did you see anything I missed? They should mostly be covered.

Reply:
`taxonId` is the only one I think -- creation source looks like it was moved over to the workflow query?

Reply:
Yeah, I didn't bother querying that one cuz it's not used by the TableRenderers.ts